scripts/hotspot.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+
+import argparse
+import csv
+import os
+import pysam
+
+def parse_command():
+    """Function to parse the command line input
+
+    Args:
+        None
+
+    Return:
+        Namespace: returns the args as a standard Namespace object
+    """
+    parser = argparse.ArgumentParser(description='Flag HotSpots in VCF')
+    parser.add_argument('--hotspots',
+        required=True,
+        help='TSV file containing the hotspots with the following format <csq-field>:<amino-acid[ref]><protein-position[start]> in the 4th column')
+    parser.add_argument('--vcf',
+        required=True,
+        help='Input VCF or VCF.GZ file')
+    parser.add_argument('--csq_field',
```

Comment:
It'd be helpful to add the possible options to the help:
```
if (field == "Gene" and not value.startswith("ENSG")) or (field == "Feature" and not value.startswith("ENST"))
```
And also to the `doc` once this becomes a tool